index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@100;400&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header>
+      <div class="nav-wrapper">
+        <nav class="navbar">
+          <div class="navtop">
+            <div class="time">
+              <h2>9:41</h2>
+            </div>
+            <div class="icons">
+              <i class="fa fa-signal" aria-hidden="true"></i>
+              <i class="fa fa-wifi" aria-hidden="true"></i>
+              <i class="fa fa-battery-full" aria-hidden="true"></i>
+            </div>
+          </div>
```

Comment:
- [ ] Please, let's try to remove this section. Your portfolio will be used on many mobile devices. You don't need to implement the time, signal, wifi, and battery icons.